docs/logagent/how-to-reduce-log-volume.md

```diff
@@ -0,0 +1,113 @@
+title: Best Practices to Reduce Log Volume with Logagent
+description: Reduce log volume to save storage or money for volume based licenses. 
+
+Centralized Logs can often be very noisy. Developers love verbose log output for potential troubleshooting. Do you want to collect all logs, all the time and pay for processing and storage? Probably not! A good practice is to reduce the log volume and focus on relevant log messages until a problem arrives. Log shipper settings can be adjusted to fetch all logs for troubleshooting and debugging if needed. Let’s have a look at the available Logagent filters for adjusting the log volume for operational needs. 
```

Comment:
Do you want to collect all logs, all the time and pay for processing and storage?
=>
Do you want to collect all logs, all the time, and pay for processing and storage?